examples/README.md

```diff
@@ -16,6 +16,9 @@
   - [Build bookdown site](#build-bookdown-site) - Build a
     [bookdown](https://bookdown.org) site and deploy it to
     [netlify](https://www.netlify.com/).
+  - [Build bookdown site, alternative](#build-bookdown-site-alternative)
+    - Build a [bookdown](https://bookdown.org) site and deploy it to
```

Comment:
This line break is an unhappy accident that causes a sub-bullet. WHY 😭

Reply:
Because it is indented? why do you have a line break here?

Reply:
I don't in the `.Rmd`. It's formatted like all the rest. But after `.Rmd` to `.md` this unfortunate accident happens.

Reply:
Oh I see, interesting. There must be some way to avoid this, but I am not sure how.